src/CMakeLists.txt

```diff
@@ -70,6 +70,10 @@ INSTALL(FILES res/oneclickinstall.png DESTINATION /usr/share/icons/hicolor/32x32
 INSTALL(FILES res/ticks-endless.gif DESTINATION /usr/share/one-click-installer/res/)
 INSTALL(FILES res/conflictCheck.gif DESTINATION /usr/share/one-click-installer/res/)
 INSTALL(FILES res/conflictIcon.png DESTINATION /usr/share/icons/hicolor/32x32/apps/)
+INSTALL(FILES res/okIcon.png DESTINATION /usr/share/icons/hicolor/32x32/apps/)
+INSTALL(FILES res/success.png DESTINATION /usr/share/icons/hicolor/32x32/apps/)
+INSTALL(FILES res/errorInstall.png DESTINATION /usr/share/icons/hicolor/32x32/apps/)
+
```

Comment:
Oh, I didn't notice those icons are not standard icons from the hicolor icon theme. I think it would be better if all those icons are installed to  /usr/share/one-click-installer/res/ instead of putting them together with standard icons. Then, you can use either hardcoded paths as you did with gif animations, or you can use QStandardPaths::locate(QStandardPaths::AppDataLocation, "res/okIcon.png") for example, to locate that file.